dags/sagemaker-run-model.py

```diff
@@ -75,13 +55,41 @@ def upload_data_to_s3(s3_bucket, test_s3_key):
                         bucket_name=s3_bucket,
                         replace=True)
 
-    upload_data = upload_data_to_s3(s3_bucket, test_s3_key)
+    upload_data = upload_data_to_s3("{{ var.value.get('s3_bucket') }}", test_s3_key)
```

Comment:
Also how does the Variable parsing work for Astro SDK? Is it at dag parse time or dag run time?

Reply:
Seems like I have not choice on avoiding the backend...

```
astro.exceptions.IllegalLoadToDatabaseException: Failing this task because you do not have a custom xcom backend set up. If you use the default XCOM backend to store large dataframes, this can significantly degrade Airflow DB performance. Please set up a custom XCOM backend (info here https://www.astronomer.io/guides/custom-xcom-backends) or set the environment variable AIRFLOW__ASTRO_SDK__DATAFRAME_ALLOW_UNSAFE_STORAGE to true if you wish to proceed while knowing the risks. 
```

@kentdanas if we are ok with adding XCOM backend setup to the guide I can go ahead and implement.